debian-resizefs.sh

```diff
@@ -5,7 +5,7 @@
 # Alternative: http://www.ivarch.com/blogs/oss/2007/01/resize-a-live-root-fs-a-howto.shtml
 
 # Check current filesystem type
-ROOT_FS_TYPE="$(sed -n -e 's|^/dev/[a-z]\+[1-9]\+ / \(ext4\) .*$|\1|p' /proc/mounts)"
+ROOT_FS_TYPE="$(sed -n -e 's|^/dev/.* / \(ext4\) .*$|\1|p' /proc/mounts)"
```

Comment:
👍

Reply:
In `[1-9]\+` what is the purpose of `\+`? Make up numbers from digits 1-9?

Reply:
I'm not great with sed but I think that is a joiner ie the match is exactly:

/dev/[any letters]+[followed by number]+followed by a space

Reply:
I think it is enough to support /dev/sda0-/dev/sda9, so without `\+`

Reply:
The breaks for LVM root volumes eg: 
```
cat /proc/mounts | grep ' / '
/dev/mapper/vg-root / ext4 rw,relatime,errors=remount-ro,data=ordered 0 0
```

Reply:
Excuse me! All the way I though `.*` is mine!

Reply:
Maybe `\S\+` would have been the best!